Chatify/Chatify/asgi.py

```diff
@@ -1,14 +1,15 @@
-#
 import os
+import django
 from django.core.asgi import get_asgi_application
 from channels.routing import ProtocolTypeRouter, URLRouter
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "Chatify.settings.dev")
+django.setup()
 import chat.routing
 
 from channels.auth import AuthMiddlewareStack
 from channels.security.websocket import AllowedHostsOriginValidator
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "Chatify.settings.dev")
-
 application = ProtocolTypeRouter(
```

Comment:
If you do changes in settings file and then code is merged in develop and i take pull of develop, it might be chances of error occure in my project ?...